package.json

```diff
@@ -92,19 +85,24 @@
     "webpack-dashboard": "^0.2.0",
     "webpack-dev-server": "^1.16.2"
   },
-  "babel": {
-    "env": {
-      "test": {
-        "plugins": [
-          [
-            "__coverage__",
-            {
-              "ignore": "**/*-spec.js"
-            }
-          ]
-        ]
-      }
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
Is there a proxy available that would return a string that depends on what was imported?  Because our fileMock just returns a hard-coded string, doesn't it?